modules/sql-engine/src/integrationTest/sql/subquery/any_all/test_correlated_any_all.test

```diff
@@ -157,15 +146,13 @@ true
 true
 NULL
 
-# Ignore https://issues.apache.org/jira/browse/IGNITE-19401
-skipif ignite3
 query T
 SELECT i=ALL(SELECT i FROM integers WHERE i=i1.i OR i IS NULL) FROM integers i1 ORDER BY i;
 ----
-NULL
-NULL
-NULL
-NULL
+true
```

Comment:
I don't understand why you change expected results here

Reply:
MariaDb and PG returns 4 nulls in this case, as was in initial test

Reply:
yes, I have the similar understanding and it looks as bug

Reply:
fill the issue, annotate